state_machine/plugs/advanced/lib/pycubed.py

```diff
@@ -80,7 +80,9 @@ def __init__(self):
 
         self.data_cache = {}
         self.filenumbers = {}
-        self.vlowbatt = 5.0
+        self.vlowbatt = 3.5
+        self.f_lowbtout = False
```

Comment:
What does this do?

Reply:
vlowbatt is the low battery threshold, the f_lowbtout is something I need to remove, that was the old timeout code.